### PR TITLE
85 highlight video when speaking

### LINKED ISSUE #85: Highlight Video when participant is speaking
MUX and other WebRTC providers give us an event when they detect a participant starts and stops speaking. We wrapped these events [here](https://github.com/montevideo-tech/hybrid-meetup-platform/blob/develop/frontend/src/lib/webrtc.js#L39).

We could listen to them and show an indicator on the Video component. The most common and easiest solution is to add a border to the video tile, while Google Meet, for example, show an animated icon in the middle of the tile.

Not sure in which milestone this belongs to, but I'd imagine probably M3.

### PR DESCRIPTION
**Pull request by gerardoquinckequalabs**
Added event listeners for participants when they join: StartedSpeaking and StoppedSpeaking.
Highlight the video box with a colored border when participant is speaking.
The video box now have rounded corners.

### CODE REVIEW

**Review by hernanr99 — APPROVED** [review]

**Review by ghbogarin — COMMENTED** [review]
@gerardoquinckequalabs everything seems to be working fine!

Small detail: I get a white line at the bottom of the video container. That was already there, but it becomes visible now that we have a border in the video container. 
I suggest adding `display: 'block'` in the style of the video tag (line 52). That seems to solve that issue.

**Review by clararfe — CHANGES_REQUESTED** [review]
Hi Gera, there is an issue when you enter a room where there is already a participant. In this case the participantJoined event is not triggered so we don't listen to the startSpeaking event. We should add this listener on the subscribeToRemoteStreams as well.

**Review by JonaQuaLabs — APPROVED** [review]

File: package-lock.json
Comment (Unnil) [inline package-lock.json:1]: Should this file be merged?

Comment (fpyoung) [reply package-lock.json:1]: you are right! we should remove this file from the PR.

Comment (fpyoung) [reply package-lock.json:1]: done!

File: frontend/src/views/Room.jsx [hunk 1]
```diff
@@ -256,6 +262,16 @@ function Room() {
...
+        p.on('StartedSpeaking', () => {
+          const streamData = remoteStreamsRef.current.get(p.id);
+          streamData.speaking = true;
+          setRemoteStreamsRef(remoteStreamsRef.current);
+        });
+        p.on('StoppedSpeaking', () => {
+          const streamData = remoteStreamsRef.current.get(p.id);
+          streamData.speaking = false;
+          setRemoteStreamsRef(remoteStreamsRef.current);
+        });
```
Comment (Unnil) [inline frontend/src/views/Room.jsx:272]: Can we encapsulate all of this in a function? I see that is duplicated when we join and then when a participant joins.

**Review by Unnil — APPROVED** [review]

File: frontend/src/views/Room.jsx (hunk 1 above)
Comment (fpyoung) [reply frontend/src/views/Room.jsx:272]: fixed!

**Pull request merged**